src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,57 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+// Class representing a User
+public class User {
+  private String id;
```

Comment:
Why is the user id a string but the recipe id and author id a long?

Reply:
because Ali says that the authentication API gives the user a String id but DataStore gives you a long id
the author id should be a String tho I'll fix that